Gruntfile.js

```diff
@@ -548,6 +555,7 @@ module.exports = function(grunt) {
 
   grunt.registerTask('unit', 'Lint and unit tests', [
     'jshint',
+    'exec:blankLinkCheck',
```

Comment:
Our other custom lint rules are in the `precommit` task which is run before CI and can be run as a commit hook. Move this there too.